libraries/core_libs/network/src/tarcap/packets_handlers/vote_packet_handler.cpp

```diff
@@ -20,6 +23,35 @@ void VotePacketHandler::validatePacketRlpFormat([[maybe_unused]] const PacketDat
   }
 }
 
+bool VotePacketHandler::isVoteRoundStepRelevant(const std::shared_ptr<Vote> &vote) const {
+  auto checking_round = pbft_mgr_->getPbftRound();
+  // If period is not the same we assuming current round is equal to 0
+  // So we won't accept votes for future period with round bigger than kVoteAcceptingSteps
+  if (pbft_chain_->getPbftChainSize() != vote->getPeriod()) {
+    checking_round = 0;
+  }
+  const auto vote_hash = vote->getHash();
+  if (vote->getRound() > checking_round + kVoteAcceptingRounds) {
+    LOG(log_nf_) << "Skip vote " << vote_hash.abridged() << ". Round too big " << pbft_mgr_->getPbftRound() << " "
```

Comment:
no for pbft_mgr_->getPbftRound(). Also it would be nice to see in log which round is related to vote and which round is current pbft round

Reply:
also https://github.com/Taraxa-project/taraxa-node/pull/1957/files#r954845005